nrm_django/nrm_site/settings/prod.py

```diff
@@ -10,4 +10,4 @@
 if database_url:
     DATABASES = {"default": dj_database_url.parse(database_url)}
 
-ALLOWED_HOSTS = ["fs-nrm.app.cloud.gov"]
+ALLOWED_HOSTS = ["fs-nrm.app.cloud.gov", "127.0.0.1"]
```

Comment:
I think that `settings/dev.py` is better to use, rather than adding this here.

Reply:
The reason I added it here was for when you want to use connect-to-service and develop locally with the cloud db. But if you feel strongly....